### ISSUE TITLE
Scroll Bars on the Horizontal Thumbnail Carousels

### ISSUE THREAD

**Issue by Pckool** [WFCD/tennotv#35]
### Tenno.tv Issue
---

**Summary (short):**
When the page is rendered, the horizontal thumbnail carousels have vertical scroll bars for some reason. Not sure why or how but it seems to be there in my environment.

---
**If issue, Reproduction:**
1. Go to the website
2. Look at the page.

---
**Mockups, screenshots, evidence:**
![0bea0bae-b1a6-43f2-a569-06c5f0920095 png](https://user-images.githubusercontent.com/10131267/50863914-5979de00-136e-11e9-9c71-13c0180ac468.jpg)

Screenshots of it going wrong.

---

**Additional context**
This may have been caused by the fix I implemented in #32, but I'm not entirely sure because I did not change anything with the container or even the height of the elements. Strange...

**Comment by TobiTenno** [comment]
I'm not seeing any scroll bars in there, also, the fix doesn't seem to have worked for me.

**Comment by Pckool** [comment]
I just checked on another system and browser and the vertical scroll bars are still there. Also I realized that what I had done does fix the issue, but only when the site is first rendered. If you navigate to a new page with a horizontal thumbnail carousels or anything of the sort.
![annotation 2019-01-08 193245](https://user-images.githubusercontent.com/10131267/50868167-4837ce00-137c-11e9-914d-6c0366624246.jpg)
